### DISCUSSION TITLE
Report "Context validation" seems to ignore text blocks if images are included

### DISCUSSION THREAD

**Discussion by mrapnx** [monarc-project/MonarcAppFO#389] [General]
Dear community,

I'm new at MONARC and tried to fill my "Context validation" report with content during "Context establishment". When I inculde an image into the "Risk analysis context" text block, the resulting report doesn't contain any content at all:
![grafik](https://user-images.githubusercontent.com/64850246/136804138-0d9e2a44-6949-4403-b901-fb316e4f2758.png)

Resulting report:
Please note that there is no text on chapter "1.1" (I used the stock template for generating the report)
![grafik](https://user-images.githubusercontent.com/64850246/136803861-ae21f499-2d67-4baa-b4a6-34d09281471a.png)

Is that behaviour intended?

Kindest regards
Marcus

**Comment by K8Sudo** [comment]
Hey @mrapnx,

it is currently not planned to attach files in the report.
see: https://github.com/monarc-project/MonarcAppFO/issues/376

**Reply by mrapnx** [reply]
Der Sudo, thanks for clarification! After removing the paperclip buttons, it's still possible to paste images into the text area. Maybe it would be a convinience to prevent that or remove pasted images / non-text content immediately after pasting.

Interesstingly, there's a strange behaviour when pasting formatted text into that field. MONARC displays it correctly, but Word displays an "unreadable content" error message. This might also benefits form a cleaning of pasted content.

Kindest regards
Marcus
